components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -374,6 +387,64 @@ void AdsServiceImpl::Start() {
   BackgroundHelper::GetInstance()->AddObserver(this);
 }
 
+void AdsServiceImpl::MaybeShowFirstLaunchNotification(bool rewards_enabled) {
+  if (!rewards_enabled || !ShouldShowAdsNotification())
+    return;
+
+  if (profile_->GetPrefs()->GetTime(
+          prefs::kBraveAdsLaunchNotificationTimestamp) == base::Time()) {
+      profile_->GetPrefs()->SetTime(
+          prefs::kBraveAdsLaunchNotificationTimestamp,
+        base::Time::Now());
+  }
+
+  auto* rewards_service =
+      brave_rewards::RewardsServiceFactory::GetForProfile(profile_);
+
+  auto* rewards_notification_service =
+      rewards_service->GetNotificationService();
+
+  RewardsNotificationService::RewardsNotificationArgs args;
+  rewards_notification_service->AddNotification(
+      RewardsNotificationService::REWARDS_NOTIFICATION_ADS_LAUNCH,
+      args,
+      rewards_notification_ads_launch);
+
+  auto timeout = base::Time::Now() - profile_->GetPrefs()->GetTime(
+      prefs::kBraveAdsLaunchNotificationTimestamp) +
+      base::TimeDelta::FromSeconds(24 * 60 * 60 * 7);
```

Comment:
We need to add new ads flag for this one, so that QA can test it. For example clear it after 5min if flag is present